internal/template/templates/.claude/skills/moai-workflow-project/templates/doc-templates/product-template.md

```
@@ -1,7 +1,7 @@
-# yoda - Mission & Strategy
+# {{PROJECT_NAME}} - Mission & Strategy
 
 ## What problem do we solve?
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)